test/hotspot/jtreg/vmTestbase/nsk/jdb/caught_exception/caught_exception002/caught_exception002.java

```diff
@@ -60,47 +60,44 @@
 
 package nsk.jdb.caught_exception.caught_exception002;
 
-import nsk.share.*;
-import nsk.share.jdb.*;
+import nsk.share.Paragrep;
+import nsk.share.jdb.JdbCommand;
+import nsk.share.jdb.JdbTest;
 
-import java.io.*;
-import java.util.*;
+import java.io.PrintStream;
+import java.util.Vector;
 
 public class caught_exception002 extends JdbTest {
 
-    public static void main (String argv[]) {
+    public static void main(String[] argv) {
         System.exit(run(argv, System.out) + JCK_STATUS_BASE);
     }
 
-    public static int run(String argv[], PrintStream out) {
-        debuggeeClass =  DEBUGGEE_CLASS;
+    public static int run(String[] argv, PrintStream out) {
+        debuggeeClass = DEBUGGEE_CLASS;
         firstBreak = FIRST_BREAK;
         lastBreak = LAST_BREAK;
         return new caught_exception002().runTest(argv, out);
     }
 
-    static final String PACKAGE_NAME       = "nsk.jdb.caught_exception.caught_exception002";
-    static final String TEST_CLASS         = PACKAGE_NAME + ".caught_exception002";
-    static final String DEBUGGEE_CLASS     = TEST_CLASS + "a";
-    static final String FIRST_BREAK        = DEBUGGEE_CLASS + ".main";
-    static final String LAST_BREAK         = DEBUGGEE_CLASS + ".lastBreak";
+    static final String PACKAGE_NAME = "nsk.jdb.caught_exception.caught_exception002";
+    static final String TEST_CLASS = PACKAGE_NAME + ".caught_exception002";
+    static final String DEBUGGEE_CLASS = TEST_CLASS + "a";
+    static final String FIRST_BREAK = DEBUGGEE_CLASS + ".main";
+    static final String LAST_BREAK = DEBUGGEE_CLASS + ".lastBreak";
```

Comment:
I'm not so sure I'd consider this change an improvement. Maybe remove some of the extra spaces but keep the alignment.

Reply:
although horizontal alignment (of variable names, initialization, expressions, etc) seems to somewhat improve readability, it almost always associated with a higher maintenance cost, and the current consensus is not to do that. from 'Horizontal Whitespace' section of the same [guidelines](http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html#toc-whitespace):
> In variable declarations it is not recommended to align types and variables.
*Motivation*
The improvement in readability when aligning variable names is negligible compared to the efforts needed to keep them aligned as the code evolves. Realigning all variables when one of the types change also causes unnecessarily complicated patches to review.

other java code guidelines either discourage horizontal alignment or consider it optional and provide the same motivation as above to why it's better not to have it.

Reply:
While in general I agree that gratuitous aligning of variables is not desirable, this is a special case where it adds value (enough that I think it's worth doing). The value here is due to the fact that the variables are building on each other. When they are aligned it's easier to scan for the variable (and its value) that another variable builds on.

You can keep your change in place if you feel that consistency and maintainabilty is more important here than readability.

Reply:
Although I don't think that the test code has the same frequency of changes as the product code, I'd still prefer consistency and maintainability over some small penalty (which is IMHO negligible) in readability. 

frankly, it's also partially motivated by my laziness as I don't want to go over all these files again and examine each and every initialization for possible (scanty) readability improvements from horizontal alignment.